blocks/hero-block/features/hero/default.jsx

```diff
@@ -0,0 +1,162 @@
+import React from "react";
+import PropTypes from "@arc-fusion/prop-types";
+
+// Arc Themes Components - Base set of components used to compose blocks
+// https://github.com/WPMedia/arc-themes-components/
+import { Button, Heading, HeadingSection, Paragraph, Stack } from "@wpmedia/arc-themes-components";
+
+const BLOCK_CLASS_NAME = "b-hero";
+
+function Hero({ customFields }) {
+	const {
+		layout = "overlay",
+		alignment = "center",
+		description,
+		headline,
+		imageURLDesktop,
+		imageURLMobile,
+		desktopMediaBreakpoint,
+		imageAltText,
+		link1Action,
+		link1Text,
+		link1Type,
+		link2Action,
+		link2Text,
+		link2Type,
+		subHeadline,
+		variant = "dark",
+	} = customFields;
+
+	const classes = [
+		BLOCK_CLASS_NAME,
+		variant === "dark" ? `${BLOCK_CLASS_NAME}--dark` : `${BLOCK_CLASS_NAME}--light`,
+	].join(" ");
+
+	return (
+		<div className={classes}>
+			<picture>
+				<source srcSet={imageURLDesktop} media={`(${desktopMediaBreakpoint})`} />
+				<img
+					className={
+						layout === "stacked"
+							? `${BLOCK_CLASS_NAME}--image-stacked`
+							: `${BLOCK_CLASS_NAME}--image`
+					}
+					src={imageURLMobile}
+					alt={imageAltText}
+				/>
+			</picture>
+
+			<Stack
+				alignment={alignment}
+				className={`${BLOCK_CLASS_NAME}--text ${BLOCK_CLASS_NAME}--${layout} ${
+					alignment === "left" ? `${BLOCK_CLASS_NAME}--left` : `${BLOCK_CLASS_NAME}--center`
+				}`}
+			>
+				<HeadingSection>
+					{headline ? <Heading>{headline}</Heading> : null}
+					{subHeadline ? (
+						<HeadingSection>
+							<Heading className={`${BLOCK_CLASS_NAME}--subheadline`}>{subHeadline}</Heading>
+						</HeadingSection>
+					) : null}
+					{description ? <Paragraph>{description}</Paragraph> : null}
+
+					{(link1Action && link1Text) || (link2Action && link2Text) ? (
+						<Stack
+							className={`${BLOCK_CLASS_NAME}--buttonrow`}
+							direction="horizontal"
+							justification={alignment === "center" ? "center" : "start"}
+						>
+							{link1Action && link1Text && (
```

Comment:
We should be using ternary to conditional render

Reply:
ternaries it is